docs/root/intro/version_history.rst

```diff
@@ -109,16 +109,16 @@ Version history
 * access log: added REQUESTED_SERVER_NAME for SNI to tcp_proxy and http
 * admin: added :http:get:`/hystrix_event_stream` as an endpoint for monitoring envoy's statistics
   through `Hystrix dashboard <https://github.com/Netflix-Skunkworks/hystrix-dashboard/wiki>`_.
-* cli: Added support for :ref:`component log level <operations_cli>` command line option for configuring log levels of individual components.
+* cli: added support for :ref:`component log level <operations_cli>` command line option for configuring log levels of individual components.
```

Comment:
Given it's now only checking the (empty) current version I could revert these, but my preference is to leave them fixed up.  Given Harvey's out for the week I may yet get away with it ;-)